content/how-to/shim.md

```diff
@@ -1,8 +1,113 @@
 ---
-title: How to Shim?
+title: How to Shim Third party libraries?
 ---
-> import-loader
-> exports-loader
-> expose-loader
-> ProvidePlugin
-> script-loader
+
+`webpack` as a module bundler can understand modules written as ES2015 modules, CommonJs or AMD. But many times, while using third party libraries, we see that they expect dependencies which are global aka `$` for `jquery`. They might also be creating global variables which need to be exported. Here we will see different ways to help webpack understand these __broken modules__.
+
+### Prefer unminified src(CommonJs/AMD) files over bundled `dist` versions.
+
+Most modules link the `dist` version in the `main` field of their `package.json`. While this is useful for most developers, for webpack it is better to alias the src version because this way webpack is able to optimize dependencies better (e.g. when using the [DedupePlugin]()). However in most cases `dist` works fine as well.
```

Comment:
Can you remove the empty link or add a stub page somewhere? Otherwise the lint will fail.